### PR TITLE
Add Saturation Module

### PR DESCRIPTION
**Pull request by danielleshwed**
Added saturation module. Inputs 0 would be black and white, 1 would be no change and 2 would be double saturation 
Normal Image
image1_0
Black and White
image1_1
Saturated
image1_1

### CODE REVIEW

File: src/ExportBin.js.orig [hunk 1]
```diff
@@ -0,0 +1,70 @@
+var fs = require('fs');
+var getDirectories = function(rootDir, cb) {
+  fs.readdir(rootDir, function(err, files) {
+      var dirs = [];
+<<<<<<< HEAD
+      if(typeof(files)==0) {
```
Comment (tech4GT) [inline src/ExportBin.js.orig]: Not correctly merged

File: src/modules/Blur/Module.js [hunk 2]
```diff
@@ -1,3 +1,61 @@
+/*
+<<<<<<< HEAD
```
Comment (tech4GT) [inline src/modules/Blur/Module.js]: Not merged correctly

File: src/modules/Blur/info.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "blur",
+    "name": "Blur",
     "description": "Blur an image by a given value",
```
Comment (tech4GT) [inline src/modules/Blur/info.json]: I did that on purpose actually all modules follow kebab-case

File: .gitignore [hunk 4]
```diff
@@ -2,6 +2,7 @@
 logs
 *.log
 npm-debug.log*
+.DS_Store
```
Comment (tech4GT) [inline .gitignore]: We already added .ds_store in .gitignore

File: src/ExportBin.js [hunk 5]
```diff
@@ -6,6 +6,12 @@ var getDirectories = function(rootDir, cb) {
         cb(dirs);
         return [];
       }
+
+      if(files.length === 0){
```
Comment (tech4GT) [inline src/ExportBin.js]: We already fixed that, see the above check

**Comment by tech4GT** [comment]
@daniellashwed great work on the module🎉
Please have a look at the requested changes
Thanks😁

File: src/ExportBin.js [hunk 6]
```diff
@@ -6,6 +6,7 @@ var getDirectories = function(rootDir, cb) {
         cb(dirs);
         return [];
       }
+
```
Comment (tech4GT) [inline src/ExportBin.js]: Please Remove this for convention sake

File: src/modules/Blur/Module.js [hunk 7]
```diff
@@ -6,39 +63,39 @@ module.exports = function Blur(options,UI){
     options.title = "Blur";
     options.description = "Blur an Image";
     options.blur = options.blur || 2
-    
+
```
Comment (tech4GT) [inline src/modules/Blur/Module.js]: The whole module.js content are written twice somehow, please look into it, my suggestion is clone our master separately, delete your blur/module.js and replace it with ours, that way no unnecessary changes would be made to it

File: src/ExportBin.js (hunk 6 above)
Comment (tech4GT) [reply src/ExportBin.js]: Sorry for being so nitpicky

**Comment by jywarren** [comment]
Hi, i just wanted to also say this looks really awesome! 👍 

Yes, the Blur module being in there would be great to resolve first, as well as compiling with `grunt build` to generate the all-in-one-file `/dist/` versions. If you need help with these steps I'm happy to pitch in! You may be able to click "Raw" on this page to get the current publiclab/master version: https://github.com/publiclab/image-sequencer/blob/master/src/modules/Blur/Module.js

Then paste it over your version to cancel out all your changes (within this PR) and push up one more commit. Then as this PR is squashed, everything on `Blur/Module.js` will just be cancelled out. Make sense?

Great work here!

**Comment by tech4GT** [comment]
@jywarren actually this is already done, please have a look at pr #193

**Comment by tech4GT** [comment]
@jywarren this pr can be closed now😁

**Comment by jywarren** [comment]
Ah, awesome!!! Cool - let's link there and close this. 👍 👍 

FWIW you can keep adding commits or even force push commits to the same PR so there isn't a strong reason to open a new one -- but either way, it's not a big deal! Thanks!!!

Moving to #193
